submissions/jeffersonMormah/CSS/style.css

```diff
@@ -0,0 +1,219 @@
+}
+header nav{
+    float: right;
+    margin-top: 15px;
+}
+.signup-content {
+    overflow: hidden;
+    padding: 40px auto;
+    margin: auto;
+    width: 90%;
+}
+main {
+    float: left;
+    text-align: left;
+    width: 45%;
+    margin-top: 55px;
+}
+main h1 {
+    font-size: 50px;
+    font-family: 'Franklin Gothic Medium';
+    color: coral;
+}
+main p  {
+    font-size: 25px;
+    font-family: 'Franklin Gothic Medium';
+    color: brown;
+}
+main a {
+    font-size: 20px;
+    font-family:'Franklin Gothic Medium';
+    color: blue;
+}
+aside {
+    float: left;
+    width: 48%;
+    margin-top: 55px;
+    margin-left: 40px;  
+}
+aside .form-document {
+    padding: 20px;
+    width: 94%;
+    border: 1px solid black;
+    border-radius: 20px;
+    margin-bottom: 12px;
+}
+.form-document label {
+    color: rgb(250,128,114);
+    font-size: 20px;
+    margin-bottom: 20px;
+    display: inline-block;
+    width: 48%;
+    margin-left: 20px;
+    font-family: 'Franklin Gothic Medium';
+}
+ .form-document input {
+     margin-top: 1%;
+     margin-left: 20px;
+     border: 2px solid #000000;
+     border-radius: 50px;
+     width: 90%;
+     height: 30px;
+ }
+ .form-document button {
+     width: 75px;
+     height: 45px;
+     border-radius: 10px;
+     display: inline-block;
+     padding: 10px;
+     background-color: coral;
+     color: black;
+     margin-left: 200px;
+ }
+ button:hover,
+ button:focus {
+     background: #0000FF;
+     transition: all 2s ease-in-out;
+     color: #F5F5DC;
+ }
+ footer {
+     overflow: hidden;;
+     padding: 20px 0 0 50px;
+     color: rgb(250,128,114);
+ }
+ .footer1, 
+ .footer2, 
+ .footer3 {
+     float: left;
+     width: 28%;
+ }
+ .footer1 ul li,
+ .footer3 ul li
+ {
+     margin-bottom: 8px;
+     font-size: 17px;
+    font-family: 'Franklin Gothic Medium';
+    font-weight: bold;
+ }
+ .footer1 ul li a{
+    display: inline;
+    color: rgb(250,128,114);  
+}
+.footer2 input[type="email"]{
+    height: 25px;
+    padding: 3px 18px 3px ;
+    font-size: 12px; 
+}
+.footer2 button[type="submit"]{
+    height: 30px;
+    padding: 0 5%;
+    font-size: 12px;
+    background-color:  rgb(250,128,114);
+    color: #F5F5DC;
+    font-weight: bold;
+}
+.footer3 ul li{
+    display: inline-flex;
+    padding:10px ;
+    margin-left: 30px;
+    margin-top: -35px;
+}
+.footer3 ul li img{
+    width: 50px;
+    text-align: left;
+}
+footer h5{
+    text-align: center;
+    margin-top: 5px;
+    font-family: 'Franklin Gothic Medium';
+    font-weight: bold;
+}
+footer p{
+    text-align: center;
+    display: block;
+    margin-top: 150px;
+    margin-left: 30px;
+}
+@media screen and (max-width:850px){
+    header #nav-head,
+    header nav,
+    header .nav-links  nav li{
+        text-align: center;
+        padding:0 3%;
+        float: none;
+}
+main, aside {
+    width: 70%;
+    margin: auto;
+    padding-bottom: 17px;
+}
+main {
+    text-align: center;
+}
+.footer1, .footer2 {
```

Comment:
@Jeffbuchi Why not add `.footer3` here as well instead of declaring it separately.